<launch>

  <arg name="model" default="$(find hw9_1)/urdf/three_link_manipulator.xacro"/>
  <arg name="gui" default="true" />
  <arg name="rvizconfig" default="$(find hw9_1)/rviz.rviz" />

  <param name="robot_description" command="$(find xacro)/xacro.py $(arg model)" />
  <param name="use_gui" value="$(arg gui)"/>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rvizconfig)" required="true" />
<!--  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />-->
  <node name="robot_state_publisher" pkg="robot_state_publisher" type="state_publisher" />
  <node name="tf_publisher" pkg="hw9_1" type="tf_publisher" />

</launch>
